src/apps/api/serializers/competitions.py

```diff
@@ -123,6 +123,7 @@ class Meta:
             'docker_image',
             'allow_robot_submissions',
             'competition_type',
+            'leaderboard_mode',
```

Comment:
leaderboard mode should be attached to either phase or leaderboard, I think? some phases may want different flags

Reply:
I recently added this to the `Leaderboard` model. I didn't realize you were currently working on this task, but we could probably move over to using these flags.
`
    ADD = "Add"
    ADD_DELETE = "Add_And_Delete"
    ADD_BY_REVIEW = "Add_By_Review"
    FORCE_LAST = "Force_Last"
    FORCE_LATEST_MULTIPLE = "Force_Latest_Multiple"
    FORCE_BEST = "Force_Best"
    SUBMISSION_RULES = (
        (ADD, "Only allow adding one submission"),
        (ADD_DELETE, "Allow users to add a single submission and remove that submission"),
        (ADD_BY_REVIEW, "Only allow adding one submission that is approved by review"),
        (FORCE_LAST, "Force only the last submission"),
        (FORCE_LATEST_MULTIPLE, "Force latest submission to be added to leaderboard (multiple)"),
        (FORCE_BEST, 'Force only the best submission to the leaderboard')
    )

`

Reply:
@ckcollab okay, sounds great

Reply:
@Logan-Ruf okay, so what i am going to do is use these flags instead of current pr flags? 👍